example/ghost/t8_test_ghost_large_level_diff.cxx

```diff
@@ -0,0 +1,233 @@
+    t8_forest_set_scheme(forest, t8_scheme_new_default_cxx() );
+    t8_forest_set_level(forest, level);
+    sc_flops_start (&fi);
+    sc_flops_snap (&fi, &snapshot);
+    t8_forest_commit (forest);
+    sc_flops_shot (&fi, &snapshot);
+    sc_stats_set1 (&stats[0], snapshot.iwtime, "New");
+    if(!no_vtk){
+        t8_forest_write_vtk(forest, "Forest_new");
+    }
+    t8_global_productionf("Successfully commited forest.\n");
+
+    //Adapt
+    t8_forest_init(&forest_adapt);
+    refine += level;
+    t8_forest_set_user_data(forest_adapt, &refine);
+    t8_forest_set_profiling(forest_adapt, 1);
+    t8_forest_set_adapt(forest_adapt, forest, t8_ghost_fractal_adapt, 1);
+    t8_forest_commit(forest_adapt);
+    if(!no_vtk){
+        t8_forest_write_vtk(forest_adapt, "Forest_adapt");
+    }
+    t8_global_productionf("Successfully refined forest adaptivly.\n");
+
+    //Partition
+    t8_forest_init(&forest_partition);
+    t8_forest_set_partition(forest_partition, forest_adapt, 0);
+    t8_forest_set_ghost_ext(forest_partition, 1, T8_GHOST_FACES, 3);
+    t8_forest_set_profiling(forest_partition, 1);
+    t8_forest_commit(forest_partition);
+    if(!no_vtk){
+        t8_forest_write_vtk(forest_partition, "Forest_partition");
+    }
+    t8_global_productionf("Successfully partitioned forest.\n");
+    t8_forest_ghost_print(forest_partition);
+
+    t8_forest_print_profile (forest_partition);
+    t8_forest_unref (&forest_partition);
+
+    sc_stats_compute (comm, 1, stats);
+    sc_stats_print (t8_get_package_id (), SC_LP_STATISTICS, 1, stats, 1, 1);
+
+
+}
+
+int
+main (int argc, char *argv[])
+{
+  int                 mpiret, parsed, dim, level, refine, mpisize, helpme, no_vtk;
+  sc_options_t       *opt;
+  const char         *path;
+  char                usage[BUFSIZ];
+  char                help[BUFSIZ];
+  t8_cmesh_t          cmesh;
+
+  snprintf (usage, BUFSIZ, "Usage:\t%s <OPTIONS> <ARGUMENTS>",
+            basename (argv[0]));
+  snprintf (help, BUFSIZ,
+            "This program reads a .msh file "
+            "created by the GMSH program and constructs a "
+            "t8code coarse mesh from them.\n\n%s\n\nExample: %s -f A1\nTo open the file A1.ms."
+            "\n\nThe default dimension of the mesh to read is 2. Since the "
+            ".msh format stores elements of all (lower) dimensions "
+            "the user must provide the argument for a different dimension by hand, if "
+            "desired.\n", usage, basename (argv[0]));
+
+  mpiret = sc_MPI_Init (&argc, &argv);
+  SC_CHECK_MPI (mpiret);
+
+  sc_init (sc_MPI_COMM_WORLD, 1, 1, NULL, SC_LP_ESSENTIAL);
+  p4est_init (NULL, SC_LP_ESSENTIAL);
+  t8_init (SC_LP_DEFAULT);
+
+  mpiret = sc_MPI_Comm_size (sc_MPI_COMM_WORLD, &mpisize);
+  SC_CHECK_MPI (mpiret);
+
+  opt = sc_options_new (argv[0]);
+  sc_options_add_switch(opt, 'h', "help", &helpme, "Display a short help message:");
+  sc_options_add_string (opt, 'p', "path", &path, "", "The path to the mesh-file. "
```

Comment:
Keep this consistent with the other examples and use "-f" and "prefix" instead of "-p" and "path".

Reply:
Do you want the int variable path also to be changed into prefix?

Reply:
yes